.github/workflows/release_mirth.yml

```
@@ -0,0 +1,66 @@
+        required: true
+        options:
+          - DEV CENTRAL
+          - PROD CENTRAL
+
+jobs:
+  publish-release:
+    runs-on: ubuntu-22.04
+    if: github.repository_owner == 'wso2'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)